diagnostic_rt_qpcr/libraries/diagnosticrtqpcrcompositions/source.rb

```diff
@@ -23,7 +23,7 @@ def build_stripwell_primer_probe_compositions(operations:)
       given_stripwells = options[:stripwell_id]
       stripwells = []
       if given_stripwells
-        stripwells = Collection.find(given_stripwells)
+        stripwells = given_stripwells.map { |id| Collection.find(id) }
```

Comment:
Why are you reverting this?

Reply:
No idea.... I didn't change it.   Something must not be up to date when I was working on this.